RFC/src/RFC-0300_DAN.md

```diff
@@ -1 +1,179 @@
+still being discussed.
+
+#### Registration
+VNs register themselves on the [Base Layer] using a special [transaction] type. The registration [transaction] type
+requires spending of a certain minimum amount of [Tari coin], the ([RegistrationCollateral]), that has a time-lock on the
+output for a minimum amount of time ([RegistrationPeriod]) as well as some metadata, such as the VNs public key and a generated Node ID. The
+details of this transaction are not confidential and are publicly auditable. The Node ID is generated during registration to prevent mining
+of VN public keys that can be used to manipulate routing on the DAN.
+
+VNs may spend this Tari back to themselves which will end their registration.
+
+Requiring nodes to register themselves serves two purposes:
+* Makes VN Sybil attacks expensive,
+* Provides an authoritative "central-but-not-centralised" registry of validator nodes from the base layer.
+
+#### Validation of instructions
+VNs are expected to manage the state of digital assets on behalf of digital asset issuers. They receive fees as reward
+for doing this.
+* Digital assets consist of an initial state plus a set of state transition rules. These rules are set by the Tari
+  protocol, but will usually provide parameters that must be specified by the [Asset Issuer].
+* The set of VNs that participate in managing state of a specific DA is called a [Committee]. A committee is selected during the asset
+issuance process and membership of the committee can be updated at [Checkpoint]s.
+* It is the VNs responsibility to ensure that every state change in a digital asset conforms to the contract's rules.
+* VNs accept digital asset [Instructions] from clients and peers. [Instructions] allow for creating, updating, and
+  expiring digital assets on the DAN.
+* VNs provide additional collateral when accepting an offer to manage an asset, which is stored in a multi-signature
+  contract on the base layer. This collateral can be taken from the VN if it is proven that the VN engaged in
+  malicious behaviour.
+* VNs participate in fraud proof validations in the event of consensus disputes (which could result in the malicious VN's
+  collateral being slashed).
+* Digital asset metadata (e.g. large images) are managed by VNs, but whether the data is considered part of the state
+  (and thus checkpointed) or out of state depends on the type of digital asset contract employed.
+
+#### Checkpoints
+VNs periodically post checkpoint summaries to the [base layer] for each asset that they are managing. The checkpoints will form an immutable
+record of the [Digital Asset] state on the base-layer. There will be two types of checkpoints:
+* An opening checkpoint will:
+  * Specify the members of the VN committee.
+  * Lock up the collateral for the committee members for this checkpoint period.
+  * Collect the fee pool for this checkpoint period from the Asset Issuer into a Multi-Sig UTXO under the control of the committee.
```

Comment:
This is the first mention of `fee pool`. It would be helpful if there was some more explanation of this concept prior to this reference

Reply:
Indeed, I moved the Fees section above this and elaborated a bit on them.